src/usb_cam.cpp

```diff
@@ -602,7 +607,8 @@ void UsbCam::stop_capturing(void)
       type = V4L2_BUF_TYPE_VIDEO_CAPTURE;
 
       if (-1 == xioctl(fd_, VIDIOC_STREAMOFF, &type))
```

Comment:
This code is coming from ROS, so they use a lot of these single line if's with no brackets. We should use brackets whenever we're writing the code, it's safer and more readable.